src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -0,0 +1,800 @@
+        "https://github.com/cortezaproject/corteza/pull/775/commits/11dccaea9f299895e5637929c02e1b0dfb518bcd"
+      ]
+    },
+    {
+      "what": [
+        "Route description field, Filters sorting, Profiler hits purging and Integration gateway settings."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/775/commits/a64f4c18b3d428bb207ab9f7f95c5a42324a9bab",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/352865b4fb9aab5d0a1697bca88987f06d4e913e"
+      ]
+    },
+    {
+      "what": [
+        "support for viewing and undeleting a deleted single or multiple records in a record list."
+      ],
+      "why": [
+        "to allow users to view and restore deleted records."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/740/commits/101c1c7e3026285052c9a731684c2cb278047144"
+      ]
+    },
+    {
+      "what": [
+        "`BETWEEN` and `NOT BETWEEN` operator support for single value fields (number and date-time module fields)."
+      ],
+      "why": [
+        "to extend record filtering support for range filtering of numeric and date-time for single value field."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/813/commits/2027409b0d8875a48fd0017264ba03f5535733ec"
+      ]
+    },
+    {
+      "what": [
+        "support for a fully customizable navigation page block with multiple navigation item type and customizable options for the navigation list items."
+      ],
+      "why": [
+        "to have the ability to show one or two level of dynamic and static navigation."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/41f1283e9d393393c651b8ac5a28cae09733f1fd"
+      ]
+    },
+    {
+      "what": [
+        "ability to drill down charts, by clicking on a data entry inside the chart.",
+        "Based on the configuration it can either filter an existing record list on the page, that is showing the records from the same module as the chart, or open a new record list in a modal."
+      ],
+      "why": [
+        "to make it easier for users to filter record lists based on chart data, in order to view the details of said data."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/877/commits/cfc4ab7424ed1a131bb34cfaed8c4ca18eef1b07"
+      ]
+    },
+    {
+      "what": [
+        "support for conditional fields."
+      ],
+      "why": [
+        "to give users more control on how records are displayed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/770/commits/1d286aff83f012e33cabc3456ffa175f7c105eb7"
+      ]
+    },
+    {
+      "what": [
+        "support for navigating to the next or previous record when inside a record page."       
+      ],
+      "why": [
+        "to allow users to navigate to previous and next record page."
```

Comment:
Perhaps a reason based on an example would be better here? to allow users go over all their leads one by one and ...; something in this style.
As is, this reads like a copy-paste of what